nipype/interfaces/setup.py

```diff
@@ -24,6 +24,7 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('mne')
     config.add_subpackage('mrtrix')
     config.add_subpackage('mrtrix3')
+    config.add_subpackage('niftyseg')
```

Comment:
I add back the base.py just to create NiftySegCommand because of the version that isn't available for all the executables.

I also edited slightly the base.py for NiftyRegCommand.
The test should pass now.

Let me know what you think.
 
Cheers,
Ben